sources/osgShader/Compiler.js

```diff
@@ -936,15 +935,16 @@ define( [
                 'uniform mat4 ProjectionMatrix;',
                 'uniform mat4 NormalMatrix;',
                 '',
-                'varying vec4 VertexColor;',
-                'varying vec3 FragNormal;',
-                'varying vec3 FragEyeVector;',
+                'varying vec4 VertexColor;',// investigate lowp
```

Comment:
ok I guess what you wanted to say is:
we should investigate different precision for varying data ....

I thought you did test and you forget to remove them. Could you adjust the comments to reflex what it means, like a todo to test stuff